app/services/get-capacity-view.js

```diff
@@ -11,7 +11,7 @@ module.exports = function (id, capacityDateRange, organisationLevel) {
   var workloadReportsPromise
 
   if (organisationalUnitType === undefined) {
-    throw new Error(organisationLevel + ' should be offender-manager, region, team or ldu')
+    throw new Error(organisationLevel + ' should be offender-manager, region, team, ldu or nps')
   }
```

Comment:
Did Simon mention that we should stop using nps? Think he said switch to hmpps? 

Maybe a separate branch to refactor that?